packages/vscode-extension/src/webview/components/UrlBar.tsx

```diff
@@ -57,7 +56,10 @@ function UrlBar({ project, disabled }: UrlBarProps) {
         <span className="codicon codicon-refresh" />
       </IconButton>
       <IconButton
-        onClick={() => project.goHome()}
+        onClick={() => {
+          project.goHome();
+          setUrlList([]);
```

Comment:
This isn't ideal but we relied on metro reload to do it for us. In the future, we need to keep `/` always in the URL list. I'll create a ticket for that.